controller/misc/config_service.py

```diff
@@ -20,14 +15,10 @@ def __get_config() -> Dict[str, Any]:
 
 
 def refresh_config():
-    response = requests.get(REQUEST_URL)
-    if response.status_code != 200:
-        raise ValueError(
-            f"Config service cant be reached -- response.code{response.status_code}"
-        )
+    response = get_config(False)
     global __config
-    __config = response.json()
-    daemon.run_without_db_token(invalidate_after, 3600)  # one hour as failsave
+    __config = response
+    daemon.run_without_db_token(invalidate_after, 3600)  # one hour as fail safe
```

Comment:
is this still necessary? The refresh of the config is for rerequesting from the config service. since the gateway is now the config service it should always be up to date